queries/atm.py

```diff
@@ -0,0 +1,571 @@
+"""
+
+    Greynir: Natural language processing for Icelandic
+
+    Atm query response module
+
+    Copyright (C) 2023 Miðeind ehf.
+
+       This program is free software: you can redistribute it and/or modify
+       it under the terms of the GNU General Public License as published by
+       the Free Software Foundation, either version 3 of the License, or
+       (at your option) any later version.
+       This program is distributed in the hope that it will be useful,
+       but WITHOUT ANY WARRANTY; without even the implied warranty of
+       MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+       GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see http://www.gnu.org/licenses/.
+
+
+    This module handles ATM-related queries.
+
+"""
+
+from typing import Any, Dict, List, Mapping, Optional, cast
+
+import os
+import logging
+import random
+import json
+
+from geo import distance
+from tree import Result, Node
+from queries import ContextDict, Query, QueryStateDict
+from reynir import NounPhrase
+from queries.util import (
+    gen_answer,
+    distance_desc,
+    krona_desc,
+    natlang_seq,
+    read_grammar_file,
+    AnswerTuple,
+    LatLonTuple,
+)
+from speech.trans.num import number_to_text
+
+_ATM_QTYPE = "Atm"
+
+_PATH_TO_ISB_JSON = "../resources/geo/isb_locations.json"
+
+_FOREIGN_CURRENCY: Mapping[str, str] = {
+    "USD": "bandaríkjadalir",
+    "GBP": "sterlingspund",
+    "EUR": "evrur",
+    "DKK": "danskar krónur",
+    "SEK": "sænskar krónur",
+    "PLN": "pólsk slot",
+    "ISK": "íslenskar krónur",
+}
+
+
+def help_text(lemma: str) -> str:
```

Comment:
This is function is not used unless you define TOPIC_LEMMAS like in some of the other modules. Maybe best to leave it out for now.

Reply:
Or add in the TOPIC_LEMMA "hraðbanki" and more examples of queries supported by this module in random help_text

Reply:
Fixed. Added a topic lemma and further query examples.